integrations/maven-bloop/src/main/scala/bloop/integrations/maven/MojoImplementation.scala

```diff
@@ -94,15 +94,28 @@ object MojoImplementation {
         emptyLauncher
       }
 
+    val scalaContext = mojo.findScalaContext()
     val compileSetup = mojo.getCompileSetup()
-    val allScalaJars = mojo.getAllScalaJars().map(abs).toList
+    val compilerAndDeps = scalaContext.findCompilerAndDependencies().asScala
+    val allScalaJars = compilerAndDeps.map { artifact =>
+      artifact.getFile().toPath()
+    }.toList
+
+    val scalaOrganization = compilerAndDeps
+      .collectFirst {
+        case artifact
+            if artifact.getArtifactId() == "scala3-compiler_3" || artifact
+              .getArtifactId() == "scala-compiler" =>
+          artifact.getGroupId()
+      }
+      .getOrElse("org.scala-lang")
```

Comment:
Just to make sure I'm following, is this just here to catch another distribution of the compiler or something? We can't just rely on `org.scala-lang`?

Reply:
We should be ok to rely on `org.scala-lang` in most cases, however they have been different compilers in the past (I think all are unused) and I don't want to change the logic here.

Reply:
👍🏼 figured just wanted to make sure.